utils/utils.go

```diff
@@ -180,7 +180,7 @@ func CreateClient(conf NetConf) (*client.Client, error) {
 	log.Infof("Configured environment: %+v", os.Environ())
 
 	// Load the client config from the current environment.
-	clientConfig, err := client.LoadClientConfig("")
+	clientConfig, err := client.LoadClientConfig(conf.DatastoreFile)
```

Comment:
I think that is a good idea too.  I already added that to the LoadClientConfig function in my changes that are in a libcalico-go PR.  I just double checked and it wasn't in the other exposed functions loading config so I added it to them too (LoadClientConfigFromBytes and LoadClientConfigFromEnvironment).